src/Mod/CAM/Path/Op/CircularHoleBase.py

```diff
@@ -25,6 +25,7 @@
 import Path
 import Path.Op.Base as PathOp
 import Path.Base.Drillable as Drillable
+import PathScripts.PathUtils as PathUtils
```

Comment:
Per the linting: `Use 'from PathScripts import PathUtils' instead (consider-using-from-import)`